src/main/java/com/sybit/airtable/Airtable.java

```diff
@@ -182,9 +186,14 @@ public void setProxy(String proxy) {
 
         this.config.setProxy(proxy);
         if (proxy == null) {
-            Unirest.setProxy(null);
+            Unirest.config().reset();
```

Comment:
It doesn't affect me since I'm not using a proxy. I just put the reset call in the setProxy method in case any other users call setProxy after calling configure and using the http client, because Unirest's Config.proxy() method calls validateClientsNotRunning() which throws an exception that says we're supposed to call reset() before setting the proxy on a running client. 

https://github.com/Kong/unirest-java/blob/main/unirest/src/main/java/kong/unirest/Config.java#L205

But I have no objection to skipping that reset() call if you think it's risky or useless.

Reply:
Yes i think the `reset()` should be done only once because otherwise the previously set values will be reset also. Because of that I suggest to move it only once time to line `151` as mentioned above.